project.yaml

```diff
@@ -7,7 +7,7 @@ schema:
   file: ./schema.graphql
 network:
   endpoint: wss://neumann.api.onfinality.io/public-ws
-  genesisHash: '0xe81cddea5dc9954ccf7aa868d231082a2bab589dd4f81592c24c490fd2e231c8'
+  genesisHash: '0x8a671b9d609a2f753cb7618f5443e301e6300e07ea85c43956d81302d88fcd81'
```

Comment:
`0x68577f68a47847fb974c220189333c46e83a899b6e375abf50b9a615d9019c20` is the new neumann endpoint

Reply:
i'll edit this